src/azure-cli/azure/cli/command_modules/acr/check_health.py

```diff
@@ -136,6 +139,7 @@ def _get_cli_version():
 # Get helm versions
 def _get_helm_version(ignore_errors):
     from ._errors import HELM_VERSION_ERROR
+    from distutils.version import LooseVersion  # pylint: disable=import-error,no-name-in-module
```

Comment:
Is it safe to disable "import-error"? Won't it break any customers using older versions of python?

Reply:
Looks like it's safe. It's been used in a few other places in the CLI codebase.

For instance:
https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/azure/cli/core/util.py#L139

https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/vm/_actions.py#L110

Reply:
Good to know it's been used in other places in CLI code itself. Should be safe then. Thanks!